src/Components/PresentationalComponents/CSAwRuleBox/NotVulnerableBox.js

```diff
@@ -51,7 +51,7 @@ const NotVulnerableBox = ({ synopsis, notVulnerableSystemCount }) => {
                                 component={TextVariants.small}
                             >
                                 <Link
-                                    to={`/cves/${synopsis}`}
+                                    to={`/insights/vulnerability/cves/${synopsis}`}
```

Comment:
In all these instances where `/insights/vulnerability` is prepended you can instead just import the `InsightsLink` component instead of the react router `Link` and keep the path without prepending anything, because the `InsightsLink` component will do that. 

What you will have to do to make the router have the correct path and the links being able to match is to remove the leading slash on every path in the `constants.js` file like here: https://github.com/RedHatInsights/vulnerability-ui/pull/1905/files#diff-178c1fd4b9b509d05dced9e21e093a5248d2ef34b3827583eb6ef1b68513c0f0R45